<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTestPlGiDialogBase</class>
 <widget class="QDialog" name="MyTestPlGiDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Test Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select input Sat image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_avail_layers"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>What type of image is this: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_sattype">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select survey shapefile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_survey"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Attribute name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Combo_LayerItem"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>MyTestPlGiDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>854</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>MyTestPlGiDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>985</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
